src/constants/tooltip.js

```diff
@@ -169,3 +169,8 @@ export const TOOLTIP_FORMATS = {
     type: TOOLTIP_FORMAT_TYPES.DATE_TIME
   }
 };
+
+export const COMPARE_TYPES = {
+  DECIMAL: 'decimal',
```

Comment:
shouldn't this be `absolute` vs `percentage`

Reply:
Yes! Lost in translation I guess.